src/Main.java

```diff
@@ -0,0 +1,363 @@
+        int[] array = new int[amountOfElemets];
+        for (int i = 0; i < amountOfElemets; i++) {
+            array[i] = (int) (Math.random() * 100);
+        }
+        return array;
+    }
+
+    public static int minElementInArray(int[] givenArray) {
+        int minElement = givenArray[0];
+        for (int i = 1; i < givenArray.length; i++) {
+            if (givenArray[i] < minElement) {
+                minElement = givenArray[i];
+            }
+        }
+        return minElement;
+    }
+
+    public static int maxElementInArray (int[] givenArray) {
+        int maxElement = givenArray[0];
+        for (int i = 1; i < givenArray.length; i++) {
+            if (givenArray[i] > maxElement) {
+                maxElement = givenArray[i];
+            }
+        }
+        return maxElement;
+    }
+
+    public static int differenceBetweenMinAndMaxElementsInArray (int [] givenArray) {
+        return maxElementInArray(givenArray) - minElementInArray(givenArray);
+    }
+
+    public static boolean comparisonElementsTwoArrays(int[] givenArray1, int[] givenArray2) {
+        if (givenArray1.length != givenArray2.length) {
+            return false;
+        }
+        else {
+            for (int i = 0; i < givenArray1.length; i++) {
+                if (givenArray1[i] != givenArray2[i]) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    public static int searchMissedNumberFromZeroToNineInArray (int[] givenArray) {
+        final int[] arrayWithNumbersFromZeroToNine = new int[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+        int missedNumber = -1;
+
+        for (int i = 0; i < arrayWithNumbersFromZeroToNine.length; i++) {
+            int counter = 0;
+            for (int j = 0; j < givenArray.length; j++) {
+                if (arrayWithNumbersFromZeroToNine[i] != givenArray[j]) {
+                     counter++;
+                     if (counter == givenArray.length) {
+                         missedNumber = arrayWithNumbersFromZeroToNine[i];
+                     }
+                }
+                else {
+                    break;
+                }
+            }
+        }
+        return missedNumber;
+
+    }
+
+    public static int[][] generateTwoDimensionalArray(int length, int width) {
+        int[][] array = new int[length][width];
+        //int number = 1;
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array[i].length; j++) {
+                array[i][j] = (int) (Math.random() * 10 - 5);
+                /*array[i][j] = number;
+                number ++;*/
+            }
+        }
+        return array;
+    }
+
+    public static void printTwoDimensionalArray(int[][] givenArray) {
+        for (int i = 0; i < givenArray.length; i++) {
+            System.out.println(Arrays.toString(givenArray[i]));
+        }
+        /*System.out.println();
+        for (int i = 0; i < givenArray.length; i++) {
```

Comment:
why you left all those comments? what for?
either use the code or delete it

Reply:
deleted